src/pkg/Microsoft.TargetingPack.Private.CoreRT/win/Microsoft.TargetingPack.Private.CoreRT.pkgproj

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <BaseLinePackageDependencies>false</BaseLinePackageDependencies>
+    <PackagePlatforms>x64;</PackagePlatforms>
+    <PackageTargetRuntime>win-corert</PackageTargetRuntime>
+    <PreventImplementationReference>true</PreventImplementationReference>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\System.Private.CoreLib\src\System.Private.CoreLib.csproj">
```

Comment:
Considered a named property rather than `..\..\..` (or `..\..` in pkgproj's without platform runtimes)?